src/snowflake/snowpark/modin/plugin/docstrings/series_utils.py

```diff
@@ -1321,6 +1321,27 @@ def quarter():
         dtype: int8
         """
 
+    def isocalendar():
```

Comment:
Why aren't we just inheriting the docstrings from the base series class?

Reply:
I'm not exactly sure, in this case our return type is the Snowpark pandas dataframe so I wanted to write a custom docstring.

Reply:
FYI: in general, if a docstring for an OSS modin class isn't specified here then its doctests won't get run.